docs/Configure-GenericAptRepo.md

```diff
@@ -0,0 +1,31 @@
+# How to configure a generic aptitude repository
+
+Tested on Ubuntu Xenial 16.04
+
+## Requirements
+Ubuntu Xenial 16.04 where the aptitude repository will be created and configured.
+
+Ubuntu Xenial 16.04 where aptitude will be configured to install packages from the previously configured repository.
+
+## On the repository machine:
+
+Make a directory containg all debian files you want to share, in this example it will be ~/deb
+```bash
+sudo apt-get install apache2 apt-transport-https dpkg-dev
+sudo cp -r ~/deb /var/www/html
+cd /var/www/html/deb
+# This will be required to be run every time you add/remove a package.
+dpkg-scanpackages -m . | sudo gzip -c > Packages.gz
+```
+
+## On the client
+```bash
+sudo sh -c  'echo "deb [trusted=yes] http://<repo ip>/deb ./" \
```

Comment:
instead of ./ -- can you put something more configurable?

Reply:
i don't think so, we would need a more complex server, depending on the word you have there, it might generate a different path location in which apt searches for packages.